Makefile

```diff
@@ -96,7 +96,8 @@ run-debug: ## Run code
 build/nginx-agent:
 	GOWORK=off CGO_ENABLED=0 go build -ldflags=${LDFLAGS} -o ./build/nginx-agent
```

Comment:
Should the update be made to the `build/nginx-agent `target?

Reply:
as discussed `build/nginx-agent` target is redundant so removing